samples/client/petstore/swift5/alamofireLibrary/PetstoreClient/Classes/OpenAPIs/OpenAPIDateWithoutTime.swift

```diff
@@ -0,0 +1,69 @@
+// OpenAPIDateWithoutTime.swift
```

Comment:
@Jonas1893 I think here this class shouldn't be generated, and this applies to the other samples to.
Can you please delete this file in all the sample projects manually and generate the sample projects again?
Also we should change one of the sample project config to use the this class.